src/pages/benefits/viewAllBenefit/ApplicationLists.tsx

```
@@ -51,24 +41,25 @@ const DetailsButton = ({ rowData }: ICellTextProps) => {
     </HStack>
   );
 };
-const ApplicantDetails: React.FC = () => {
+
+const ApplicationLists: React.FC = () => {
   const [searchTerm, setSearchTerm] = useState("");
   const [sortOrder, setSortOrder] = useState<string>("asc");
   const { id } = useParams<{ id: string }>();
   const [applicationData, setApplicationData] = useState<any[]>([]);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)